_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+Somehow also great if held upright for perusers of vertical-video. Or, great for any user unconcerned with 90° viewing angles in either direction.
+In the glory that is NewPipe 0.20.0, a full-screen button replaces the orientation-switch button. Is the video a good old-fashioned video, or some vertical nonsense? The player knows, and the player checks to see if the device is held the same way before going full-screen. If you have auto-rotate on, the button vanishes for regular videos, but will be to go from vertical video details, to full vertical video on the vertical. If you are into that sort of thing.
 
 <video src="/img/smart_button.webm" poster="/img/0.20.0_thumbnail_smart_fullscreen.png" class="img-responsive no-flow screenshot-image" autoplay loop muted controls></video>
 
 ### Minimized player
 
-You can now swipe down on the video in the video details page to _minimize_ it - just like YouTu... wait no. It's better. You get a minimized player at the bottom, and you can play/pause or even close it from there. How cool is that? The video will keep playing (or stay paused, if you prefer that) while you browse NewPipe at your leisure! No more having to finish a video before browsing mindlessly! Yay distractions! (In landscape mode, you can swipe down on the title to minimize as well. The swipe area is confined to that region because the rest is used for the existing volume and brightness gestures, as you know.)
+Swipe down from the video details to minimize. This gives you the cutest little video, from which you can play, pause, or close down this little rascal pulling on your heartstrings. Browse away while the video stays put, or carries on. Without distractions like these, how do you really know the video you are watching is the best possible distraction from something else? In widescreen distraction, you can swipe down on the title to minimize. Other swipage in other areas still controls volume, brightness and gestures.
 <video src="/img/landscape_swipe_down.webm" poster="/img/0.20.0_thumbnail_swipe_down.png" class="img-responsive" autoplay loop muted controls></video>
-Since the player can now be minimized instead of closed, this means you get to retain your opened video queue! Let us go through an example: You watch a video, minimize it, then browse the trending page (you insane person), watch a video from there, quickly minimize it, browse for some calming music, and finally watch a video about pandas. As long as the minimized player isn't closed (by using the cross button or swiping down on it), you can tap back to go through those videos in reverse order! This allows you to browse a whole bunch of videos and quickly build up a video queue before you start watching them. You were able do this earlier, but you had to use the background or popup player queues as a workaround, since the main player's queue isn't accessible outside of the player.
```

Comment:
Trending page into insane person is a cracking joke. It works. Run with it.
I don't care as much for how to achieve what is now fixed in a version that is now the old version, because, nobody cares. Really.

Reply:
> Trending page into insane person is a cracking joke.

All thanks to @Poolitzer!

> what is now fixed

You and the user should care because this is not a bug fix. It is an entirely new feature. Unless we explain it with an example, the user will not know how it works. It is not intuitive, building a queue in reverse, or that the backstack remains attached to the minimised player. Moreover, actions related to this interfere with previously expected behaviour. We have already had to explain it to users while the PR was in development and I expect we'll still have to for some time after the release.

Reply:
Not the example, this:
> You were able do this earlier, but you had to use the background or popup player queues as a workaround, since the main player's queue isn't accessible outside of the player.

And now you will have to explain it to me what that passage does. I have read it 5 times. You can now use two methods to do the same thing? Is the point to make that the hacky way no longer works? It doesn't say that, and I have fully removed it.